local-cli/server/server.js

```diff
@@ -13,7 +13,11 @@ const formatBanner = require('./formatBanner');
 const path = require('path');
 const runServer = require('./runServer');
 const findSymlinksPaths = require('./findSymlinksPaths');
-const NODE_MODULES = path.resolve(__dirname, '..', '..', '..');
+
+const NODE_MODULES =
+  /node_modules/.test(__dirname) ?
```

Comment:
I'm not sure a regex is necessary here. I find regexes hard to read - e.g. I had to look up just now whether this means "equals" or "contains". `__dirname.indexOf('node_modules') > -1`  is definitely more ugly and verbose but probably more obvious?